Sources/SwiftCoAP/SCMessage.swift

```diff
@@ -159,7 +159,9 @@ public final class SCCoAPUDPTransportLayer {
                 return
             }
             if let error = maybeError {
-                self.notifyDelegatesAboutError(for: connection.endpoint, error: error)
+                if error != NWError.posix(.ECANCELED) {
```

Comment:
This one is neat. Never thought about getting it and not getting canceled state first.